oaknational/Oak-Web-Application
# Fix/lesq 323/downloads error handling

## Description

Provides the user with feedback when the downloads api fails. (eg. without internet access)

## How to test

1. Go to https://deploy-preview-1925--oak-web-application.netlify.thenational.academy/teachers-2023/programmes/geography-secondary-ks4/units/test-ingest-this-unit/lessons/lesson-3-in-ingest-this-unit/downloads
2. In developer tools, set throttling to offline. Click on Download Zip
3. You should see the error message rendered 

## Screenshots

How it used to look (delete if n/a):

How it should now look:
<img width="1010" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/1030540/03a2854e-7351-4cf1-8e4f-f3acca6e1673">
